core/vision/src/main/java/com/arcrobotics/ftclib/vision/UGBasicHighGoalPipeline.java

```diff
@@ -54,9 +55,24 @@ public UGBasicHighGoalPipeline() {
         redRect = new Rect();
 
         minThreshold = 155;
-        maxThreshold = 255;
+        maxThreshold = 200;
+
```

Comment:
Make the braces (top and bottom) flush with the method body. No whitespace. i.e.
```java
method header {
  stuff
  // single spaces in between lines in the method body is fine
  stuff

  stuff
}
```